script.js

```diff
@@ -212,12 +212,27 @@ formemail.addEventListener('submit', (e) => {
   }
 });
 
-// Local storage section
+const clientName = document.getElementById('fullname');
+const clientEmail = document.getElementById('email');
+const clientMessage = document.getElementById('message-input');
+const formKey = 'formemail';
+
+window.onload = () => {
+  const formData = localStorage.getItem(formKey);
+
+  if (formData) {
+    const form = JSON.parse(formData);
+    clientName.value = form.clientName;
+    clientEmail.value = form.clientEmail;
+    clientMessage.value = form.clientMessage;
+  }
+};
+
```

Comment:
- Please just as the previous reviewer explained, the form stored to the local storage but does not retrieve.  

**Hint:** 

The error is coming from your line 221 `const formData = localStorage.getItem(formKey);` 

Try this code to fix it and request for another review (You can reach the Code reviewer on slack  `@kanu mike chibundu`)

```javascript
window.onload = () => {
  const formData = localStorage.getItem('formKey');

  if (formData) {
    const form = JSON.parse(formData);
    clientName.value = form.clientName;
    clientEmail.value = form.clientEmail;
    clientMessage.value = form.clientMessage;
  }
};

formemail.addEventListener('input', () => {
  const formstorage = {
    clientName: clientName.value,
    clientEmail: clientEmail.value,
    clientMessage: clientMessage.value,
  };
  localStorage.setItem('formKey', JSON.stringify(formstorage));
});
```

- What I changed was on line 229  from `const formData = localStorage.getItem(formKey);
` to   `const formData = localStorage.getItem('formKey');` as the const variable `formKey' is not known on reload, it would be next you do not save the local storage key in a variable for this project since it's not a large project, Use a string instead.

Happy coding :+1: